README.md

```diff
@@ -1,4 +1,5 @@
-# wavefront-sdk-go [![build status][ci-img]][ci] [![Go Report Card][go-report-img]][go-report] [![GoDoc][godoc-img]][godoc]
+# wavefront-sdk-go
+[![build status][ci-img]][ci] [![Go Report Card][go-report-img]][go-report] [![GoDoc][godoc-img]][godoc]
```

Comment:
This changed in https://github.com/wavefrontHQ/wavefront-sdk-go/commit/d7d8613002f99fefc2f796dbf22b56d381cfad07. Please be sure the changes in that commit are preserved when you rebase.